src/reactor.cc

```diff
@@ -112,12 +112,24 @@ void Reactor::Tick() {
     // record the last time series entry if the reactor was operating at the
     // time of retirement.
     if (exit_time() == context()->time()) {
-      if (cycle_step > 0 && cycle_step <= cycle_time &&
+      if (refuel_time == 0){
+        if (cycle_step > 0 && cycle_step <= cycle_time &&
           core.count() == n_assem_core) {
         cyclus::toolkit::RecordTimeSeries<cyclus::toolkit::POWER>(this, power_cap);
       } else {
         cyclus::toolkit::RecordTimeSeries<cyclus::toolkit::POWER>(this, 0);
       }
+      }
```

Comment:
our style guide prefers to put else on the same line as the curly brace that closes the if statement.